app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+
+    def find_human(name: str) -> Person:
+        return next((human for human in person_list if human.name == name), None)
```

Comment:
The `find_human` function is defined inside `create_person_list`, which is fine for encapsulation, but if you need to use this function elsewhere, consider moving it outside to make it more reusable.